app/src/main/java/de/westnordost/streetcomplete/quests/curb_ramps_tactile_pavings/CheckCurbRamps_TactilePavings.java

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.curb_ramps_tactile_pavings;
+
+import android.os.Bundle;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class CheckCurbRamps_TactilePavings extends SimpleOverpassQuestType
+{
+	@Inject public CheckCurbRamps_TactilePavings(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		//return "nodes, ways with "(("footway"="crossing") "and" ("kerb"="lowered"))";
+
+		return "nodes, ways with (" +
```

Comment:
This should be `ways with (kerb=lowered and !tactile_paving)`, or similar

Reply:
Not displaying for some reason. `nodes with kerb=lowered and !tactile_paving` might also work. Debug for alternative explanations as well - maybe not finding the icon, etc?

Reply:
Also use a good icon!